src/renderer/components/pool/PoolDetails.tsx

```diff
@@ -64,10 +66,12 @@ export const PoolDetails: React.FC<Props> = ({
         <A.Col xs={24} md={16}>
           <ChartView priceRatio={priceRatio} />
         </A.Col>
-      </Styled.TopContainer>
-      <A.Col span={24}>
-        <HistoryView poolAsset={asset} />
-      </A.Col>
+      </Styled.Section>
+      <Styled.Section>
+        <A.Col span={24}>
+          <HistoryView poolAsset={asset} />
+        </A.Col>
+      </Styled.Section>
```

Comment:
looks like this `Styled.Section` is e redundant. here is example of how i would implement it in this case
![image](https://user-images.githubusercontent.com/67793194/120234052-cbb12d00-c25f-11eb-89c2-0a0b00a66982.png)
![image](https://user-images.githubusercontent.com/67793194/120234061-d4096800-c25f-11eb-951d-6edf4e218d26.png)

```diff
diff --git a/src/renderer/components/pool/PoolDetails.style.tsx b/src/renderer/components/pool/PoolDetails.style.tsx
index 3236568c..356ca1a2 100644
--- a/src/renderer/components/pool/PoolDetails.style.tsx
+++ b/src/renderer/components/pool/PoolDetails.style.tsx
@@ -3,11 +3,11 @@ import styled from 'styled-components'
 import { Asset, Chain } from '@xchainjs/xchain-util'
-import * as A from 'antd'
 import BigNumber from 'bignumber.js'
 import * as O from 'fp-ts/Option'
 
@@ -41,37 +40,31 @@ export const PoolDetails: React.FC<Props> = ({
   const price = useMemo(() => H.getPrice(poolDetail, priceRatio), [poolDetail, priceRatio])
   return (
     <Styled.Container>
-      <Styled.Section>
-        <A.Col span={24}>
-          <PoolTitle
-            network={network}
-            haltedChains={haltedChains}
-            asset={O.some(asset)}
-            price={price}
-            priceSymbol={priceSymbol}
-            isLoading={isLoading}
-          />
-        </A.Col>
-      </Styled.Section>
-      <Styled.Section>
-        <A.Col xs={24} md={8}>
-          <PoolCards
-            poolStatsDetail={poolStatsDetail}
-            priceRatio={priceRatio}
-            poolDetail={poolDetail}
-            priceSymbol={priceSymbol}
-            isLoading={isLoading}
-          />
-        </A.Col>
-        <A.Col xs={24} md={16}>
-          <ChartView priceRatio={priceRatio} />
-        </A.Col>
-      </Styled.Section>
-      <Styled.Section>
-        <A.Col span={24}>
-          <HistoryView poolAsset={asset} />
-        </A.Col>
-      </Styled.Section>
+      <Styled.Col span={24}>
+        <PoolTitle
+          network={network}
+          haltedChains={haltedChains}
+          asset={O.some(asset)}
+          price={price}
+          priceSymbol={priceSymbol}
+          isLoading={isLoading}
+        />
+      </Styled.Col>
+      <Styled.Col xs={24} md={8}>
+        <PoolCards
+          poolStatsDetail={poolStatsDetail}
+          priceRatio={priceRatio}
+          poolDetail={poolDetail}
+          priceSymbol={priceSymbol}
+          isLoading={isLoading}
+        />
+      </Styled.Col>
+      <Styled.Col xs={24} md={16}>
+        <ChartView priceRatio={priceRatio} />
+      </Styled.Col>
+      <Styled.Col span={24}>
+        <HistoryView poolAsset={asset} />
+      </Styled.Col>
     </Styled.Container>
   )
 }

```

Reply:
Thank you for the thoroughness, using Ant's Grid Layout system instead of adding a custom styled component `Section` makes sense. PR updated